packages/styleguide/src/components/Chart/Table.js

```diff
@@ -254,18 +284,107 @@ Table.propTypes = propTypes
 export default Table
 
 const Cell = (props) => {
-  const { type, width, color, colorScale, value, children, isNumeric } = props
+  const {
+    columnName,
+    width,
+    color,
+    colorScale,
+    value,
+    children,
+    isNumeric,
+    isBarColumn,
+    barChartExtent,
+    numberFormatter,
+  } = props
+
+  // const [parentWidth, setParentWidth] = useState(null)
+  // const parentCell = useCallback((node) => {
+  //   if (node !== null) {
+  //     setParentWidth(node.getBoundingClientRect().width)
+  //   }
+  // })
+
+  const parentWidth = Math.max(width, 100)
+
+  const barScale = scaleLinear()
+    .domain(barChartExtent)
+    .range([5, parentWidth - PADDING])
```

Comment:
why start at `5`? Should zero be zero pixel? If you want to ensure `0.1` is at least 1 px IMO you could `Math.ceil` the `barWidth` after the scale.